pom.xml

```diff
@@ -693,5 +695,14 @@
 				</repository>
 			</repositories>
 		</profile>
+		<profile>
+			<id>doclint-java8-disable</id>
```

Comment:
@spohnan This is what was breaking javadoc on jdk7 - the -Xdoclint:none  blows up there (it was just collapsed automatically by travis in the install phase block in the log).   This profile should only activate on jdk8+ - javadoc not looks for ${javadoc.opts}, so it's either empty (jdk7), or has the doclint string in it.